frameworks/helloworld/tests/test_region_awareness.py

```diff
@@ -14,18 +14,45 @@
 POD_NAMES = ['hello-0', 'world-0', 'world-1']
 
 
-@pytest.mark.dcos_min_version('1.11')
-@pytest.mark.region_awareness
-@sdk_utils.dcos_ee_only
-def test_nodes_deploy_to_local_region_by_default():
+@pytest.fixture
+def local_service():
+    try:
+        sdk_install.install(
+            config.PACKAGE_NAME,
+            config.SERVICE_NAME,
+            3,
+            additional_options={
+                "service": {"scenario": "MULTI_REGION", "allow_region_awareness": "true"}
+            })
+        sdk_plan.wait_for_completed_deployment(config.SERVICE_NAME)
+
+        yield
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.fixture
+def remote_service():
     sdk_install.install(
         config.PACKAGE_NAME,
         config.SERVICE_NAME,
         3,
-        additional_options={"service": {"scenario": "MULTI_REGION"}})
-
+        additional_options={
+            "service": {
+                "scenario": "MULTI_REGION", "allow_region_awareness": "true", "region": "Europe"
+            }
+        })
     sdk_plan.wait_for_completed_deployment(config.SERVICE_NAME)
 
+    yield
+
+    sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.mark.dcos_min_version('1.11')
+@pytest.mark.region_awareness
+@sdk_utils.dcos_ee_only
+def test_nodes_deploy_to_local_region_by_default(local_service):
     for pod_name in POD_NAMES:
         info = sdk_cmd.service_request(
```

Comment:
I still think that a function:
```python
def get_pod_region(service_name: str, pod_name: str) -> bool:
    r = sdk_cmd.service_request('GET', service_name, '/v1/pod/{}/info'.format(pod_name))

    info = r.json()[0]['info']

    return [l['value'] for l in info['labels']['labels'] if l['key'] == 'offer_region'][0]
```
would give us some utility. For example:
```python
    for pod_name in POD_NAMES:
        pod_region = get_pod_region(config.SERVICE_NAME, pod_name)
        expected_region = "Europe"
        assert pod_region == expected_region, "Region mismatch for pod {}. " \
                                              "Expected: {} Found: {}".format(pod_name, expected_region, pod_region)
```

Some notes:
* Should the "USA" and "Europe" regions be global constants (`LOCAL_REGION` and `REMOTE_REGION`)?
* Where does the default of "USA" come from? Is this part of the cluster definition?
* What behaviour do we expect when the pod info has:
    * No region label? (Currently we're going to get an index out of bounds exception
    * Multiple region labels?